.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!-- 
+Please ensure that you complete the following mandatory sections:
+
+- Issue*
+- Overview
+- Reason*
+- Work carried out
+- Confirmations
+
+* Only mandatory if working from a issue
+ -->
+
+## 🧰 Issue
+<!-- [The issue the work was done for as an issue reference (e.g. `closes #1`)] -->
+
+## 🚀 Overview: 
+<!-- [A summary of what you did in no more than one paragraph] -->
+
+## 🔗 Link to preview (or screenshot, if relevant)
+<!-- [If you're on a project which auto-deploys branches, link to the branches preview link here] -->
+
+## 🤔 Reason: 
+<!-- [Why did you do what you did? - This should be copied from the User Story part of the issue if it is available] -->
+
+## 🔨Work carried out:
+
+<!-- [A list of work you have done, use [markdown checklist format](https://github.blog/2013-01-09-task-lists-in-gfm-issues-pulls-comments/), if you leave any boxes unchecked, be sure to leave this PR as a draft. If the issue has Acceptance Criteria, you should include those items to show that you have accomplished the goals of the issue ] -->
+
+- [x] Tests pass
+
+## 🖥️ Screenshot
+<!-- [If the work is UI related then paste a screenshot of the update here. Where possible, please use an animated screenshot.] -->
+
+## Confirmations
+
+- [ ] I have chosen reviewers for my PR.
+- [ ] I have assigned myself to this PR.
```

Comment:
Can you _assign_ yourself to a PR? Or do you mean that you've assigned yourself to the issue linked to the PR?

Reply:
I know.  It surprised me when I saw it on another project.  _Assign_ is offered beneath reviewers here in the Pull Request.

Let me learn more about assigning a PR.

Reply:
Ok, there isn't really a definition for what a PR assignee is.  Since it's me that does the merge, and it's the person that submitted the PR that is the one that resolves any issues, I don't thin we need it.

I'll delete that line.